patches/minecraft/net/minecraft/client/renderer/RenderTypeLookup.java.patch

```diff
@@ -87,6 +92,12 @@
 +      setRenderLayer(block, type::equals);
 +   }
 +
++   public static void setRenderLayerWithFabulous(Block block, RenderType type) {
++      java.util.Objects.requireNonNull(type);
++      java.util.function.Predicate<RenderType> predicate = type::equals;
++      setRenderLayer(block, predicate.or(t -> t.equals(Atlases.func_239280_i_())));
```

Comment:
This will break if a mod already uses `setRenderLayer` right?

Reply:
Not sure what you mean

Reply:
If my mod calls `RenderTypeLookup.setRenderLayer(MyBlocks.MY_BLOCK, e -> e == RenderType.getTransparent())`
then calls `RenderTypeLookup.setRenderLayerWithFabulous(MyBlocks.MY_BLOCK, Atlases.func_239280_i_())`
`setRenderLayerWithFabulous` will overwrite my  `setRenderLayer` call. Just store these in a seperate map instead of reusing setRenderLayer's map.

Reply:
okay was not sure why that would be a problem, but it could if you want a predicate other than equals ? If thats the case i can just add another overload no?